app/uk/gov/hmrc/exports/models/Page.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models
+
+import play.api.libs.json.{Json, OFormat}
+import play.api.mvc.QueryStringBindable
+
+case class Page(index: Int = 1, size: Int = 50)
+
+object Page {
+  implicit val format: OFormat[Page] = Json.format[Page]
+  implicit def bindable(implicit intBinder: QueryStringBindable[Int]): QueryStringBindable[Page] =
```

Comment:
If you haven't seen these before it means we can use complex types in `app.routes` and  Controller params and it "just works"

I expect Notifications will want this eventually too

Reply:
I don't think we have to take implicit argument here - this `QueryStringBindable[Int]` could be field in `QueryStringBindable[Page]` with usage of `implicitly` language function

Reply:
Nice

Reply:
Fixed